tasmota/tasmota_xdrv_driver/xdrv_01_9_webserver.ino

```diff
@@ -2315,7 +2315,7 @@ void HandleOtherConfiguration(void) {
     if (i == EMUL_HUE) { i++; }
 #endif
     if (i < EMUL_MAX) {
-      WSContentSend_P(PSTR("<input id='r%d' name='b2' type='radio' value='%d'%s><b>%s</b> %s<br>"),  // Different id only used for labels
+      WSContentSend_P(PSTR("<label><input id='r%d' name='b2' type='radio' value='%d'%s><b>%s</b> %s</label><br>"),  // Different id only used for labels
```

Comment:
I'm not sure what the "Different id only used for labels" comment refers to. Did not find where the ids would be used, and there were no labels before this PR.

Reply:
Why would we need to enclose all this in `<label>`?